indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -49,27 +55,35 @@ class Calendar extends React.Component {
         actions: PropTypes.exact({
             fetchCalendar: PropTypes.func.isRequired,
             setDate: PropTypes.func.isRequired,
-            fetchRoomDetails: PropTypes.func.isRequired
+            fetchRoomDetails: PropTypes.func.isRequired,
+            setFilterParameter: PropTypes.func.isRequired
         }).isRequired,
         pushState: PropTypes.func.isRequired,
+        filters: PropTypes.object.isRequired,
+        hasFavoriteRooms: PropTypes.bool,
```

Comment:
props coming from redux should usually be marked as required since they will always be present (that way you also don't need to put them in defaultProps)